tools/babel-plugin-import-d3/src/index.js

```diff
@@ -37,14 +37,6 @@ function findD3RelatedPackageName(importDeclarationPath) {
 	return found;
 }
 
-function getLastNameInPath(path) {
```

Comment:
you should release it with a note: remove dead code and a simple fix bump to keep track of changes